include/labwc.h

```diff
@@ -114,6 +115,13 @@ struct seat {
 
 	struct wlr_pointer_constraint_v1 *current_constraint;
 
+	/*
+	 * Tracks the top-most surface underneath a tablet tool when the
+	 * tool is in proximity.
+	 * Updating happens on tablet proximity and tablet axis events.
+	 */
+	struct wlr_surface *tablet_tool_surface;
```

Comment:
note to self: might also track the tool here instead, may be that allows a cleaner plumbing of the tablet functions into the general cursor logic.